.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx --no-install commitlint --edit "$1"
+npx --no-install commitlint --config commitlint.config.cjs --edit "$1"
```

Comment:
added this line because of the error.
ref. https://github.com/cypress-io/cypress/issues/28432